barretenberg/cpp/src/barretenberg/ecc/groups/affine_element.hpp

```diff
@@ -20,9 +20,9 @@ template <typename Fq, typename Fr, typename Params> class alignas(64) affine_el
 
     constexpr affine_element(const Fq& a, const Fq& b) noexcept;
 
-    constexpr affine_element(const affine_element& other) noexcept;
+    constexpr affine_element(const affine_element& other) = default;
 
```

Comment:
Just some cleanup. Rule of thumb: Unless you're using raw pointers, `default` is right

Reply:
We should run some numbers and/or double check with Zac that this is alright. Is the point that `default` is not `noexcept`?

Reply:
I can run sanity numbers, but we don't use noexcept everywhere, I'm sure it was just the linter saying it's good to have on the move operators